summarize/src/main.rs

```diff
@@ -52,20 +65,26 @@ fn main() -> Result<(), Box<std::error::Error>> {
     let total_time = results.total_time.as_nanos() as f64;
 
     for query_data in results.query_data {
-        table.add_row(row![
-            query_data.label,
-            format!("{:.2?}", query_data.self_time),
-            format!("{:.3}", ((query_data.self_time.as_nanos() as f64) / total_time) * 100.0),
-            format!("{}", query_data.invocation_count),
-            format!("{}", query_data.number_of_cache_hits),
-            format!("{:.2?}", query_data.blocked_time),
-            format!("{:.2?}", query_data.incremental_load_time),
-        ]);
+
+        let percent = (query_data.self_time.as_nanos() as f64) / total_time * 100.0;
+
+        if percent > percent_above {
+            table.add_row(row![
+                query_data.label,
+                format!("{:.2?}", query_data.self_time),
+                format!("{:.3}", percent),
+                format!("{}", query_data.invocation_count),
+                format!("{}", query_data.number_of_cache_hits),
+                format!("{:.2?}", query_data.blocked_time),
+                format!("{:.2?}", query_data.incremental_load_time),
+            ]);
+        }
     }
 
     table.printstd();
 
     println!("Total cpu time: {:?}", results.total_time);
+    println!("Showing results for % total time greater than {:?}%", percent_above);
```

Comment:
Maybe show the total percentage of the shown rows too. If for example 90% of the time is spent in tasks each less than 1% time, then it can be confusing why optimizing the shown tasks doesnt yield amy improvement.

Reply:
I agree. I think if `percent_above != 100.0`, then we should show a message:

```
Filtered results account for x% of total time
```

Reply:
You mena `percent_above != 0.0`?

Reply:
Woops. Yes, if the filtering is active (`percent_above != 0.0`), then we should show this message, otherwise it is redundant.